docroot/themes/humsci/humsci_basic/templates/menus/macros/nav-menu.twig

```diff
@@ -1,6 +1,6 @@
 {#
 /**
- * Macro for creating nested menus.
+ * Macro for creating nested menus in the main navigation.
```

Comment:
in `docroot/themes/humsci/humsci_basic/templates/menus/menu--main.html.twig` can you remove the 'no-js' class that is getting added?